express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+let nextId = 1;
+const gradesArray = [];
```

Comment:
The expected data type for the grades is an object, like we've been using in the prior exercises. The instructions actually say "empty object", not "empty array".

I realize that this particular exercise is easier with an object, but for consistency (and so you can reuse it in the notes-api exercise), you should use the same object type across these exercises.